src/utils/__tests__/lesson-metadata.test.ts

```diff
@@ -1,135 +1,167 @@
 import {LessonResource} from 'types'
-import {mergeLessonMetadata} from '../lesson-metadata'
-
-test('top-level data from Sanity overrides graphql', () => {
-  const graphqlMetadata = {
-    title: 'Ultimate React',
-    duration: 123,
-  } as LessonResource
-  const sanityMetadata = {
-    title: 'Xtreme React',
-    path: '/path/to/lesson',
-  } as LessonResource
-
-  const expectedResult = {
-    title: 'Xtreme React',
-    duration: 123,
-    path: '/path/to/lesson',
-  }
-
-  const result = mergeLessonMetadata(graphqlMetadata, sanityMetadata)
-
-  expect(result).toEqual(expectedResult)
-})
+import {mergeLessonMetadata, deriveDataFromBaseValues} from '../lesson-metadata'
 
-test('Sanity tags override graphql tags when present', () => {
-  const graphqlMetadata = {
-    tags: [{name: 'Vue'}],
-  } as LessonResource
+describe('mergeLessonMetadata()', () => {
+  test('top-level data from Sanity overrides graphql', () => {
```

Comment:
Most of the diff in this file is indenting existing tests under this `describe` block.